partnership/partnership.go

```diff
@@ -238,52 +244,72 @@ func facilePartnership(jwtData string, policy *models.Policy, product *models.Pr
 	var (
 		person models.User
 		asset  models.Asset
+		claims FacileClaims
```

Comment:
I'd remove this var declaration and leave it to the current line 252.
Otherwise would add here the declaration for the error. (Option 1 preferred)